framework/src/main/java/org/checkerframework/framework/stub/AnnotationFileParser.java

```diff
@@ -1284,7 +1298,8 @@ private void annotate(AnnotatedTypeMirror type, List<AnnotationExpr> annotations
             if (annoMirror != null) {
                 type.replaceAnnotation(annoMirror);
             } else {
-                stubWarnNotFound("Unknown annotation: " + annotation);
+                // TODO: Be more specific: In what file?
```

Comment:
The field `filename` can be used for this.  Maybe all AnnotationFileParser warnings should report the filename?